experiments.py

```diff
@@ -63,10 +69,7 @@ def run_experiments(self):
         minimax_config = MinimaxConfiguration(6)
         self.players = [
             MctsPlayer(mcts_config), 
-            Ucb1TunedPlayer(mcts_config), 
```

Comment:
probably it wasn't intentional delete

Reply:
Nope, forgot to rollback change after experiments. Done!